README.md

```diff
@@ -2,6 +2,55 @@
 
 An operator to deploy and operate cloud.redhat.com resources for Openshift.
 
+## Overview
+
+The Clowder operator takes much of the heavy lifting out of creating and 
+maintaining applications on the Clouddot platform. Instead of an app developer
+being responsible for authoring multiple resources and combining them into a
+single k8s template, the Clowder app defines a simple `ClowdApp` resource
+which not only defines the pods for the application, but also requests certain
+key resources from the environment, such as Object Storage Buckets, a Database,
+Kafka Topics, of an In Memory Database.
+
+A `ClowdEnvironment` resource is used to define how key resources, such as
+Kafka Topics and Databases are deployed. Using different providers, a single
+`ClowdEnvironment` can radically alter the way in which resources are
+provisioned. For example, with regards to Kafka Topics, setting the `provider`
+to `local` will instruct Clowder to deploy a local Zookeeper/Kafka pod
+and create topics inside it, but if the provider were set to `operator`, then
+Clowder would instead drop a KafkaTopic Custom Resource in a local ready for
+the Kafka Strimzi operator to pick up and create topics in a production-esque
+instance.
+
+The diagram below shows how the two Clowder resources are used to create all
+other k8s resources.
+
+![Clowder Flow](images/clowder-flow.svg "Clowder Flow")
+
+Once these custom resources have been created and deployed to the k8s
+environment, the operator will create a secret with all necessary configuration
+data and expose it to the pods by mounting it as a JSON. In this way, instead
+of an app configuring itself, the app is configured instead by Clowder.
+
+This has several key advantages including, but not limited to:
+
+* Consistency of deployments
+* Simpler app developer experience
```

Comment:
Having just two bullets makes me think it would be better to just format it as one or two sentences.